src/main/scala/pureconfig/ConfigConvert.scala

```diff
@@ -246,7 +246,9 @@ trait LowPriorityConfigConvertImplicits {
 
   implicit val readString = fromString[String](identity)
   implicit val readBoolean = fromString[Boolean](_.toBoolean)
-  implicit val readDouble = fromString[Double](_.toDouble)
+  implicit val readDouble = fromString[Double]({
+    v => if (v.last == '%') v.dropRight(1).toDouble / 100.0 else v.toDouble
```

Comment:
This `v.last` will fail with
`java.util.NoSuchElementException: next on empty iterator`
if `v` is an empty string. Do you think we should make an effort to provide a better error message?